FrameworkSystem/DB/ProxyDB.py

```diff
@@ -379,7 +376,7 @@ def __storeProxy(self, userDN, chain, proxyProvider=None):
       result = Registry.getProxyProvidersForDN(userDN)
       if not result['OK']:
         return result
-      proxyProvider = result.get('Value') and result['Value'][0] or 'Certificate'
+      proxyProvider = result['Value'][0] if result.get('Value') else 'Certificate'
```

Comment:
This has different behaviour.
with `result = S_OK([''])` the original returns 'Certificate', the new one `''`
```
>>> result = S_OK([''])
>>> result.get('Value') and result['Value'][0] or 'Certificate'
'Certificate'
>>> result['Value'][0] if result.get('Value') else 'Certificate'
''
```